ENH: Add details about ET setup (#99)
TheAxonLab/hcph-sops: docs/data-collection/pre-session.md

    |:--:|
    | ![start-session-syncbox](../assets/images/start-session-syncbox.png) |
    | ![run-session-syncbox](../assets/images/trigger_box.jpg) |

## Eye Tracker and projector setup

> Please note that some of the images used in this section are sourced from the [Eye Tracker Instruction Manual](https://www.sr-research.com/support/attachment.php?aid=1376)

Reviewer: ```suggestion
Please note that some of the images used in this section are sourced from the [Eye Tracker Instruction Manual](https://www.sr-research.com/support/attachment.php?aid=1376).
```

Reviewer 2: I would suggest to upload the PDF here, so that we can always access it even if the website goes down.

> !!! warning "To ensure accurate performance and prevent minor drifts in the pupil and CR thresholds, perform the ET Setup 10-15 minutes after powering the ET on."

Reviewer: The reader does not know at this point what is the CR threshold. Please introduce the abbreviation and explain (maybe earlier if there is somewhere else where this info would fit better in).

    | ![projector](../assets/images/projector.png) | ![paper-projector](../assets/images/paper-projector.png) | ![adjust-projector](../assets/images/adjust-projector.png) |
    |:--:|:--:|:--:|
    | *The hole is the part through which you should check the quality of the projection* {: colspan=3} | | |

- [ ] Verify that the projector projects your laptop screen by looking through the window of the console room.
- [ ] Open the PCT experiment in *Psychopy* (`task-pct_bold.psyexp` file).
    - [ ] Run the experiment by pressing the green play button. 
    - [ ] Press enter to begin the *camera setup mode*.
    - [ ] The camera setup mode contains a view of one eye, and you can switch that view between two modes: one is the field-of-view of the ET, the second is an automatic zoom on the eye itself (or a random part if the eye is not visible).

>     - [ ] To ease the setup of the ET, switch to the field-of-view mode in the camera setup mode by pressing the left or right arrow. This will allow you to adjust the ET position as the infrared camera is providing feedback inside the scanning room through the projector.

Reviewer: ```suggestion
    - [ ] To ease the setup of the ET, switch to the field-of-view mode in the camera setup mode by pressing :arrow_left: or :arrow_right:. This will allow you to adjust the ET position as the infrared camera is providing feedback inside the scanning room through the projector.
```

Reviewer 2: If emojies are not properly rendered, use unicode characters

Reviewer 2: I think it's better with our "keypress" style, start new sentences with newlines:
```suggestion
    - [ ] To ease the setup of the ET, switch to the field-of-view mode in the camera setup mode by pressing the <span class="keypress">◀</span> or <span class="keypress">▶</span> arrow.
        This will allow you to adjust the ET position as the infrared camera provides feedback inside the scanning room on the projection screen.
```

> - [ ]  Enter the scanning room and adjust the camera's position until the eye is clearly visible.

Reviewer: ```suggestion
- [ ]  Enter the scanning room and adjust the camera's position until the eye is clearly visible. The optimal position corresponds to where the eye is visible at the center of the square defined by the coil opening.
```

Reviewer 2: ```suggestion
- [ ]  Enter the scanning room and adjust the camera's position until the eye is fully visible.
    To adjust the camera position, slightly unscrew the fixation locks of the camera arm, move the arm in search of the correct position, and tighten the locks again on the desired position.
    The **right** eye should be visible at the center of the coil's square window for the eye.
```

> - [ ]  If the image appears blurry, fine-tune the focus using the lens barrel. Refer to the example image below.

Reviewer 2: ```suggestion
- [ ]  If the image appears blurry, fine-tune the focus using the lens barrel.
    Refer to the example image below.
```

  ![focus_example](../assets/images/focus.png)

> - [ ] Press the A key to automatically set the pupil and CR detection thresholds.

Reviewer: ```suggestion
- [ ] Press the `A` key to set the pupil and CR detection thresholds automatically.
```

Reviewer 2: We actually have some HTML to style keys:
```suggestion
- [ ] Press the <span class="keypress">A</span> key to set the pupil and CR detection thresholds automatically.
```

> - [ ] Fine-tune the pupil detection threshold using the up and down arrow keys if necessary.

Reviewer: ```suggestion
- [ ] Fine-tune the pupil detection threshold using the :arrow_up: and :arrow_down: keys if necessary.
```

Reviewer 2: ```suggestion
- [ ] Fine-tune the pupil detection threshold using the <span class="keypress">▲</span> and <span class="keypress">▼</span> arrow keys if necessary.
```

(same before)

> While checking the pupil/CR image and thresholding, ask the participant to look at the screen corners to verify correct detection even in extreme positions.
>   ![pupil_threshold](../assets/images/pupil_threshold.png)

Reviewer 2: This belongs in the previous bullet point.

```suggestion
    While checking the pupil/CR image and thresholding, ask the participant to look at the screen corners to verify correct detection even in extreme positions.
      ![pupil_threshold](../assets/images/pupil_threshold.png)
```
